### ISSUE TITLE
Don't work after updated to 1.2.5c

### ISSUE THREAD

**Issue by SnowyJune973** [jennyem/pathfinder-mods#11]
[EldritchArcana] Failed to apply patch EldritchArcana.UIUtilityUnit_CollectClassDeterminators_Patch: System.ArgumentException: No target method specified for class EldritchArcana.UIUtilityUnit_CollectClassDeterminators_Patch (declaringType=Kingmaker.UI.Common.UIUtilityUnit, methodName =CollectClassDeterminator, methodType=, argumentTypes=(Kingmaker.Blueprints.Classes.BlueprintProgression, Kingmaker.UnitLogic.UnitProgressionData))
  at Harmony12.PatchProcessor.PrepareType () [0x001de] in <455e5c1456634e14910c281b79c6c325>:0 
  at Harmony12.PatchProcessor..ctor (Harmony12.HarmonyInstance instance, System.Type type, Harmony12.HarmonyMethod attributes) [0x00065] in <455e5c1456634e14910c281b79c6c325>:0 
  at EldritchArcana.Main.ApplyPatch (System.Type type, System.String featureName) [0x0007e] in <747853739bac4da2867b221972530edd>:0

**Comment by SnowyJune973** [comment]
Guess the related method is deprecated in the 1.2.5c update?

**Comment by elmindra** [comment]
thanks for reporting this! I'm at work today but I'll take a look once I get home.

**Comment by elmindra** [comment]
FYI -- this error doesn't affect anything really. If you open the Unity Mod Manager settings, you can see the patch is for "Crossblooded bloodline icon in spellbook UI". So it's just a bit of visual polish.

**Comment by ALFAIthildur** [comment]
Is  the current version of the mod working fine with 1.2.5 then?

**Comment by elmindra** [comment]
Yes, it's working fine. The error is caused by something that isn't needed anymore.

(I had patched one game method, but the devs fixed it, so now the patch doesn't apply.)

**Comment by ALFAIthildur** [comment]
Hopefully this is the same issue/nonissue, occuring with game version 1.2.6... Nearly identical blurb from ouputlog.txt file, but string names seem to be different:

[EldritchArcana] Failed to apply patch EldritchArcana.UIUtilityUnit_CollectClassDeterminators_Patch: System.ArgumentException: No target method specified for class EldritchArcana.UIUtilityUnit_CollectClassDeterminators_Patch (declaringType=Kingmaker.UI.Common.UIUtilityUnit, methodName =CollectClassDeterminator, methodType=, argumentTypes=(Kingmaker.Blueprints.Classes.BlueprintProgression, Kingmaker.UnitLogic.UnitProgressionData))
  at Harmony12.PatchProcessor.PrepareType () [0x001de] in <455e5c1456634e14910c281b79c6c325>:0 
  at Harmony12.PatchProcessor..ctor (Harmony12.HarmonyInstance instance, System.Type type, Harmony12.HarmonyMethod attributes) [0x00065] in <455e5c1456634e14910c281b79c6c325>:0 
  at EldritchArcana.Main.ApplyPatch (System.Type type, System.String featureName) [0x0006a] in <4a5f30d73e034397a307ff24399c20af>:0
